### ISSUE TITLE
Remove "+n more" authors button within tables

### ISSUE THREAD

**Issue by junxini** [chanzuckerberg/cryoet-data-portal#572]
**Context:** To streamline user navigation and encourage click-through to the next level of data, we are removing redundant buttons. the "+n more" button opens up the metadata panel which is already covered in the "more info" button, hence not needed.

**Tasks**
- remove the "+n more" button next to the author names on the following pages:
  - browse all page
  - single run page 

**Dependency**
- #524 needs to be implemented before we can remove the button 

**Link to design**: https://www.figma.com/file/tKMoeyLbzQDLtlyrEzs9uX/Kevin---Data-Discovery-Improvements-V1?type=design&node-id=4585-104623&mode=design&t=IiLun97AOqy4hyEC-0
